### PR TITLE
[15.0][ADD] account_move_line_analytic_group

### PR DESCRIPTION
**Pull request by adriresu**

### CODE REVIEW

**Comment by adriresu** [comment]
Previous PR here:
https://github.com/solvosci/slv-account/pull/36

File: account_move_line_analytic_group/hooks.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+def pre_init_hook(cr):
+    openupgrade.logged_query(
+        cr, "ALTER TABLE account_move_line ADD COLUMN IF NOT EXISTS analytic_account_group_id INT"
+    )
+    openupgrade.logged_query(
+        cr,
+        """
+            UPDATE account_move_line
+            SET analytic_account_group_id = COALESCE(subquery.group_id, NULL)
```
Comment (dalonsod) [inline account_move_line_analytic_group/hooks.py]: You should also add root group initialization (good luck with PostgreSQL split, I don't know if exists)

File: account_move_line_analytic_group/views/account_move_line_views.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="product_template_view_form_clun_back_data" model="ir.ui.view">
+        <field name="name">account.move.line.search (Account Move Line Filter)</field>
+        <field name="model">account.move.line</field>
+        <field name="inherit_id" ref="account.view_account_move_line_filter"/>
+        <field name="arch" type="xml">
+            <xpath expr="//search/field[@name='account_id']" position="after">
+                <field name="analytic_account_group_id" groups="analytic.group_analytic_accounting"/>
+            </xpath>
```
Comment (dalonsod) [inline account_move_line_analytic_group/views/account_move_line_views.xml:10]: Could you add search for group root?

**Review by dalonsod — CHANGES_REQUESTED** [review]
Looks good, only two pending things!

**Review by dalonsod — APPROVED** [review]
Ready!

**Pull request merged**